Library/Homebrew/formula_auditor.rb

```diff
@@ -52,14 +52,16 @@ def audit_style
 
     def audit_file
       if formula.core_formula? && @versioned_formula
+        unversioned_name = formula.name.gsub(/@.*$/, "")
+
+        # ignore when an unversioned formula doesn't exist after an explicit rename
+        return if formula.tap.formula_renames.key?(unversioned_name)
+
+        # build this ourselves as we want e.g. homebrew/core to be present
+        full_name = "#{formula.tap}/#{unversioned_name}"
```

Comment:
Do we want to get rid of the `if formula.tap` check?

Reply:
It's not functionally necessary because it's inside an if statement that requires `formula.core_formula?` to be true and that means that `formula.tap` must not be nil.

https://github.com/Homebrew/brew/blob/f765382cb6d312e3cdebd955a2601998390fe0b0/Library/Homebrew/formula.rb#L1943-L1947

Reply:
Ah, of course. Sounds fine then!